packages/amplify-cli/src/commands/delete.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
   name: 'delete',
   run: async context => {
+    if ((context.parameters.array || []).length > 0) {
```

Comment:
`[].length > 0` is always `false`, I think the expression here is awkward. 

You might try:
```js
if (Array.isArray(context.parameters.array) && context.parameters.array.length) {
```

I modified that by taking the inverse of this SO answer:
https://stackoverflow.com/questions/24403732/how-to-check-if-array-is-empty-or-does-not-exist

Reply:
You're right, it is awkward. I'll swap this statement out for your suggestion.